### PR TITLE
Techdebt/362 update exercise

### PR DESCRIPTION
**Pull request by dsantoro**

### CODE REVIEW

File: frontend/src/pages/Exercise/index.js [hunk 1]
```diff
@@ -94,93 +94,82 @@ const FormContainer = styled('div')({
...
     overflow-y: scroll !important;
   }
 `;
 
-export class Exercise extends Component<Props> {
-  constructor(props: Props) {
-    super(props);
-    this.handleSubmit = this.handleSubmit.bind(this);
-  }
+function Exercise(props) {
```
Comment (bendiggle) [inline frontend/src/pages/Exercise/index.js]: Wheres the Prop type? Could also do your favourite object destructuring here 😉

File: frontend/src/pages/Exercise/index.js [hunk 2]
```diff
@@ -94,93 +94,82 @@ const FormContainer = styled('div')({
...
+  const [item, setItem] = useState();
```
Comment (bendiggle) [inline frontend/src/pages/Exercise/index.js]: Add initial state

File: frontend/src/pages/Exercise/index.js [hunk 3]
```diff
@@ -94,93 +94,82 @@ const FormContainer = styled('div')({
...
     });
   };
 
-  handleSubmit = (event: SyntheticInputEvent<*>) => {
+  const handleSubmit = (event: SyntheticInputEvent<*>) => {
     event && event.preventDefault();
 
     logAction({
-      participantId: this.props.participantId,
+      participantId: participantId,
```
Comment (bendiggle) [inline frontend/src/pages/Exercise/index.js]:
```suggestion
      participantId,
```

**Comment by fericsepi** [comment]
Issue #362

**Pull request merged**